docs/usage.md

```diff
@@ -57,27 +57,37 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
+Note that passing the reference sheet to `--fasta` to the pipeline will _override_ the corresponding user-build indices parameters (`--fasta_fai`, `--fasta_dict`).
+
+An example of a reference sheet in `csv` format is as follows:
 
 ```txt
 reference_name,fasta,fai,dict,mapper_index,circular_target,mitochondrion
+Mammoth_MT_Krause,//<path>/<to>/data/Mammoth_MT_Krause.fasta,/<path>/<to>/data/Mammoth_MT_Krause.fasta.fai,/<path>/<to>/data/Mammoth_MT_Krause.dict,/<path>/<to>/data/,NC_007596.2,NC_007596.2
+Human_MT,/<path>/<to>/data/Human_MT.fasta.gz,,,,,
 ```
 
-Only `reference_name`, and `fasta` are required.
+Only `reference_name`, and `fasta` are mandatory, whereas all other entries can be missing depending on your context.
+
+| Header          | Required | Description                                                                                                                |
+| --------------- | -------- | -------------------------------------------------------------------------------------------------------------------------- |
+| reference_name  | Yes      | Name of the reference to be used in file names and nextflow console log                                                    |
+| fasta           | Yes      | Path to FASTA of reference. Can be optionally gzipped                                                                      |
+| fai             | No       | Optional path to pre-build SAMtools `fai` index file corresponding to the FASTA                                            |
+| dict            | No       | Optional path to pre-build picard `dict` index file corresponding to the FASTA                                             |
+| mapper_index    | No       | Optional path to _directory_ containing pre-build mapper index files corresponding to the FASTA                            |
+| circular_target | No       | Optional string, and only required for CircularMapper, with name of entry in FASTA to extend (up to first space in header) |
+| mitocondrion    | No       | Optional string, and only required for MTNucRatio, with name of entry in FASTA of mitochondrion entry's header.            |
+
 Files for `fai`, `dict`, `mapper_index` will be generated by the pipeline for you if not specified.
-`circular_target` and `mitochondiron` are only needed if circular mapper and MtNucRatio modules are selected.
 
 A filled-in example is like so, where a user-supplied `.dict` file nor circular generator not MtNucRatio is required:
 
-```txt
-reference_name,fasta,fai,dict,mapper_index,circular_target,mitochondrion
-tannerella_forsythia,~/data/reference/tannerella_forstythia.fasta,~/data/reference/tannerella_forstythia.fasta.fai,,~/data/reference/tannerella_forstythia.dict,~/data/reference/,,
-```
+> ⚠️ The names of the files in the `mapper_index` directory must have the same basename as the FASTA itself! e.g. if FASTA is Mammoth_MT_Krause.fasta, then each BWA index file should be Mammoth_MT_Krausef.fasta.<suffix>
```

Comment:
the `<suffix>` does not get rendered in the final markdown document.

Reply:
Escaped